packages/w3up-client/src/client.js

```diff
@@ -291,9 +301,102 @@ export class Client extends Base {
         )
       }
     }
+
+    // Authorize the listed Content Serve Services to serve content from the created space
+    if (options.skipContentServeAuthorization !== true) {
+      if (
+        !options.authorizeContentServeServices ||
+        options.authorizeContentServeServices.length === 0
+      ) {
+        throw new Error(
+          'failed to authorize Content Serve Services: missing <authorizeContentServeServices> option'
+        )
+      }
+
+      if (!options.connection) {
+        throw new Error(
+          'failed to authorize Content Serve Services: missing <connection> option'
+        )
+      }
+
+      for (const service of options.authorizeContentServeServices) {
+        await this.authorizeContentServe(space, {
+          audience: service,
+          connection: options.connection,
+        })
+      }
+    }
+
     return space
   }
 
+  /**
+   * Authorizes an audience to serve content from the provided space and record egress events.
+   * It also publishes the delegation to the content serve service.
+   * Delegates the following capabilities to the audience:
+   * - `space/content/serve/*`
+   *
+   * @param {import('./types.js').OwnedSpace} space - The space to authorize the audience for.
+   * @param {object} options - Options for the authorization.
```

Comment:
Options are optional. Otherwise param or config.

```suggestion
   * @param {object} [options] - Options for the authorization.
```